packages/components/psammead-most-read/src/list.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import styled from 'styled-components';
+import { node, shape, string, arrayOf } from 'prop-types';
+import { EasternArabic, WesternArabic } from '@bbc/psammead-locales/numerals';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Grid from '../../psammead-grid';
+import { MostReadRank, MostReadItem } from './item';
+
+const StyledOl = styled.ol.attrs({
+  role: 'list',
+})`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+`;
+
+const StyledLi = styled.li.attrs({
+  role: 'listitem',
+})`
+  margin: 1rem;
+  padding: 0;
+  background: #99ffff;
+`;
+
+const serviceNumerals = {
+  news: WesternArabic,
+  arabic: EasternArabic,
+};
+
+const MostReadListItem = ({ service, script, item, rank }) => (
+  <StyledLi>
+    <Grid
```

Comment:
We can declare this `column` props at the top of this file to make it more readable.
see https://github.com/bbc/simorgh/blob/12ef63160f3ae04ec6fa0463d3bcd3300a934350/src/app/components/Grid/index.stories.jsx#L5

Reply:
Fixed, would like some suggestions on naming though 🤣